tests/nfs/test_file_ops_renames.py

```diff
@@ -0,0 +1,186 @@
+import concurrent.futures
+
+from ceph.waiter import WaitUntil
+from cli.ceph.ceph import Ceph
+from cli.exceptions import OperationFailedError
+from utility.log import Log
+
+log = Log(__name__)
+
+timeout, interval = 60, 2
+
+
+def create_files(client, mount_point, num_files):
+    for i in range(1, num_files + 1):
+        try:
+            client.exec_command(
+                sudo=True,
+                cmd=f"dd if=/dev/urandom of={mount_point}/file{i} bs=1 count=1",
+            )
+        except Exception:
+            raise OperationFailedError(f"failed to create file file{i}")
+
+
+def create_dirs(client, mount_point, num_dirs):
+    for i in range(1, num_dirs + 1):
+        try:
+            client.exec_command(
+                sudo=True,
+                cmd=f"mkdir {mount_point}/dir{i}",
+            )
+        except Exception:
+            raise OperationFailedError(f"failed to create directory dir{i}")
+
+
+def rename_files(client, mount_point, num_files):
+    for i in range(1, num_files + 1):
```

Comment:
It will take num_files * timeout secs in case of failed operation. To avoid this move this for loop within waiter loop